stable/memcached/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 2.7.1
+version: 2.7.2
```

Comment:
As this adds a new feature, the minor version should be bumped.

Reply:
Fixed to version `2.8.1`